steps.py

```diff
@@ -0,0 +1,9 @@
+def stepcounter():
```

Comment:
The issue is that you're defining the global variable steps inside the function - but you're never actually running the function. And when you import from this file, you're importing the global variable that never gets initialized. 

I'm also a bit confusing what this function is doing. Every time you run it, it sets steps to 0 and then adds 1. So it will only ever be 1. I think what you're trying to have is a function that runs basically anytime someone does anything - so that it keeps track of the steps. I'll leave a suggestion below!